Gruntfile.js

```diff
@@ -518,7 +531,7 @@ var overrides      = require('./core/server/overrides'),
         // Unit tests do **not** touch the database.
         // A coverage report can be generated for these tests using the `grunt test-coverage` task.
         grunt.registerTask('test-unit', 'Run unit tests (mocha)',
-            ['test-setup', 'mochacli:unit']
+            ['knex-migrator', 'test-setup', 'mochacli:unit']
```

Comment:
Ok, if we move knex-migrator into test-setup it will get run here anyway, can fix later.